### PR TITLE
Onkyo: Added support for TX-NR737

### PR DESCRIPTION
**Pull request by stasee**
Added support for TX-NR737

Added the IDs and tested with my receiver.

[onkyo] Added support for TX-NR737

Signed-off-by: stasee <ste@stasee.de>

### CODE REVIEW

**Comment by stasee** [comment]
This a quite trivial commit. Just added the support for my TX-NR737. Everything works like the TX-NR747.

**Comment by martinvw** [comment]
@paulianttila could you quickly check it? Thanks!

File: addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/tx-nr737.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="TX-NR737">
+		<label>Onkyo TX-NR737 AV Receiver</label>
+		<description>Network enabled Onkyo AV Receivers</description>
+
+		<channel-groups>
+			<channel-group typeId="zone1Controls" id="zone1" />
+			<channel-group typeId="zone2Controls" id="zone2" />
+			<channel-group typeId="playerControls" id="player" />
```
Comment (paulianttila) [inline addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/tx-nr737.xml]: TX-NR737 should have 3 zones as TX-NR727? TX-NR747 have only 2 zones.

**Comment by martinvw** [comment]
Thanks all of you!

**Pull request merged**